src/public/peoplePicker.ts

```diff
@@ -0,0 +1,97 @@
+import { ensureInitialized, isAPISupportedByPlatform } from '../internal/internalAPIs';
+import { FrameContexts } from './constants';
+import { ErrorCode, SdkError } from './interfaces';
+import { validatePeoplePickerInput } from '../internal/mediaUtil';
+import { sendMessageToParent } from '../internal/communication';
+
+export namespace peoplePicker {
+  /**
+   * This is the SDK version when people picker API is supported on mobile.
+   */
+  export const peoplePickerRequiredVersion = '2.0.0';
+
+  /**
+   * Launches a people picker and allows the user to select one or more people from the list
+   * If the app is added to personal app scope the people picker launched is org wide and if the app is added to a chat/channel, people picker launched is also limited to the members of chat/channel
+   * @param callback Returns list of JSON object of type PeoplePickerResult which consists of AAD IDs, display names and emails of the selected users
+   * @param peoplePickerInputs Input parameters to launch customized people picker
+   */
+  export function selectPeople(
+    callback: (error: SdkError, people: PeoplePickerResult[]) => void,
+    peoplePickerInputs?: PeoplePickerInputs,
+  ): void {
+    if (!callback) {
+      throw new Error('[people picker] Callback cannot be null');
+    }
+    ensureInitialized(FrameContexts.content, FrameContexts.task);
```

Comment:
Why only these two contexts? What about settings?

Reply:
Media APIs, location APIs etc and just initializing these two hence followed suit. Please let me know if I should add settings for those as well.